cmd/kpng/main.go

```diff
@@ -96,13 +94,3 @@ func setupGlobal() (ctx context.Context) {
 
 	return
 }
-
-func versionCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:   "version",
-		Short: "print the version and quit",
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println(version)
```

Comment:
This was the only reference to `fmt` in this file. However, for some reason, I couldn't remove its import. Even importing it with `_` for side effects did not work. Not sure why.